### PR TITLE
Add $VAL (Valeria) Token To prices model.

### PR DESCRIPTION
**Pull request by nathanglb**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by nathanglb** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by nathanglb** [comment]
recheck

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1622,7 +1622,8 @@ FROM
     ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e, 18),
     ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
     ('pepe-pepe', 'ethereum', 'PEPE', 0x6982508145454Ce325dDbE47a25d4ec3d2311933, 18),
-    ('wld-worldcoin', 'ethereum', 'WLD', 0x163f8c2467924be0ae7b5347228cabf260318753, 18)
+    ('wld-worldcoin', 'ethereum', 'WLD', 0x163f8c2467924be0ae7b5347228cabf260318753, 18),
+    ('val-valeria', 'ethereum', '$VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1626]:
```suggestion
    ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18)
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @nathanglb 

API: https://api.coinpaprika.com/v1/coins/val-valeria

**Comment by nathanglb** [comment]
Thanks @Hosuke - the PR said it was out of date with `main` so I merged main into my fork/branch.  The Github CI is running again now.  Thank you for the support.

**Pull request merged**